gax/src/main/java/com/google/api/gax/rpc/BatchedFuture.java

```diff
@@ -44,4 +44,14 @@
   public static <T> BatchedFuture<T> create() {
     return new BatchedFuture<>();
   }
+
+  @Override
+  public boolean set(ResponseT value) {
```

Comment:
Please add TODO and some documentation to make these protected in the future (if appropriate)